test/test_autograd.py

```diff
@@ -4239,8 +4254,6 @@ def check(name):
                         output_variable = getattr(self_variable, name)(*args_variable, **kwargs_variable)
                         output_tensor = getattr(self_tensor, name)(*args_tensor, **kwargs_variable)
                         if not isinstance(output_tensor, torch.Tensor) and not istuple(output_tensor):
-                            # TODO: I'm not sure why we insert an outer dimension
```

Comment:
Just out of curiosity, what was the answer?

Reply:
that case is executed when output_tensor is not a tensor or tuple, but a scalar and so that line just makes it a tensor